packages/harmony/src/assets/styles/border-radius.css

```diff
@@ -8,5 +8,5 @@ root {
   --harmony-border-radius-m: 8px;
   --harmony-border-radius-l: 12px;
   --harmony-border-radius-xl: 16px;
-  --harmony-border-radius-2xl: 320px;
```

Comment:
320px is comical
![image](https://github.com/AudiusProject/audius-protocol/assets/2358254/c666595b-c890-4b53-8d03-968b501e5372)